Cargo.toml

```diff
@@ -13,10 +13,11 @@ members = [
     "node",
     "runtime",
     "integration-tests",
+    "scripts/fund-dev-accounts",
```

Comment:
Perhaps exclude this one as well? It now always compiles the script when compiling the node binary which includes subxt.